docs/tutorials/01_algorithms_introduction.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
Did this tutorial already exist? Or it's new content?

Reply:
this is from qiskit-tutorials, not a history-preserving copy, but I thought we could do that in a follow-up.